package.json

```diff
@@ -30,46 +30,48 @@
     "build-storybook": "build-storybook"
   },
   "peerDependencies": {
-    "prop-types": "^15.5.4",
+    "prop-types": "^15.7.2",
     "react": "^15.0.0 || ^16.0.0",
     "react-dom": "^15.0.0 || ^16.0.0"
   },
   "devDependencies": {
-    "@rollup/plugin-typescript": "^2.1.0",
-    "@storybook/addon-actions": "^5.3.7",
-    "@storybook/addon-info": "^5.3.17",
-    "@storybook/addon-knobs": "^5.3.17",
-    "@storybook/addon-links": "^5.3.7",
-    "@storybook/addons": "^5.3.7",
-    "@storybook/cli": "^5.3.7",
-    "@storybook/preset-create-react-app": "^1.5.2",
-    "@storybook/react": "^5.3.7",
-    "@svgr/rollup": "^2.4.1",
-    "@types/jest": "^23.3.14",
-    "@types/node": "^13.9.5",
-    "@types/react": "^16.3.13",
-    "@types/react-dom": "^16.0.5",
+    "@rollup/plugin-babel": "^5.2.1",
+    "@rollup/plugin-commonjs": "^16.0.0",
+    "@rollup/plugin-node-resolve": "^10.0.0",
+    "@rollup/plugin-typescript": "^6.1.0",
+    "@rollup/plugin-url": "^5.0.1",
+    "@storybook/addon-actions": "^6.0.28",
+    "@storybook/addon-docs": "^6.0.28",
+    "@storybook/addon-knobs": "^6.0.28",
+    "@storybook/addon-links": "^6.0.28",
+    "@storybook/addons": "^6.0.28",
+    "@storybook/cli": "^6.0.28",
+    "@storybook/preset-create-react-app": "^3.1.4",
+    "@storybook/preset-typescript": "^3.0.0",
+    "@storybook/react": "^6.0.28",
+    "@svgr/rollup": "^5.4.0",
+    "@testing-library/react": "^11.1.0",
+    "@types/faker": "^5.1.4",
+    "@types/jest": "^26.0.15",
+    "@types/node": "^14.14.6",
+    "@types/react": "^16.9.55",
+    "@types/react-dom": "^16.9.9",
     "babel-core": "^6.26.3",
-    "babel-loader": "^7.1.5",
     "babel-runtime": "^6.26.0",
-    "cross-env": "^5.1.4",
-    "gh-pages": "^1.2.0",
-    "jest": "^24.9.0",
-    "mini-css-extract-plugin": "^0.9.0",
-    "prettier": "2.0.2",
+    "cross-env": "^7.0.2",
+    "faker": "^5.1.0",
+    "gh-pages": "^3.1.0",
+    "mini-css-extract-plugin": "^1.2.1",
+    "prettier": "2.1.2",
     "react": "^16.4.1",
+    "react-docgen-typescript-loader": "^3.7.2",
```

Comment:
It looks like this package is no longer being used.

Reply:
Great catch!  It must have been deprecated.